src/main/java/appeng/container/slot/AppEngSlot.java

```diff
@@ -123,15 +123,14 @@ public void putStack( final ItemStack par1ItemStack )
 	@Override
 	public void onSlotChanged()
 	{
-		if( this.inventory instanceof AppEngInternalInventory )
-		{
-			( (AppEngInternalInventory) this.inventory ).markDirty( this.getSlotIndex() );
-		}
-		else
-		{
-			super.onSlotChanged();
-		}
-
+		// TODO: should not be needed
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)